cmd/testwrapper/testwrapper.go

```diff
@@ -1,62 +1,187 @@
 // Copyright (c) Tailscale Inc & AUTHORS
 // SPDX-License-Identifier: BSD-3-Clause
 
-// testwrapper is a wrapper for retrying flaky tests, using the -exec flag of
-// 'go test'. Tests that are flaky can use the 'flakytest' subpackage to mark
-// themselves as flaky and be retried on failure.
+// testwrapper is a wrapper for retrying flaky tests, it is meant to be called
+// in place of 'go test'. It assumes the first argument is the package to test,
+// and the rest of the arguments are passed to 'go test'.
```

Comment:
so you can't pass args to go test itself like "go test -v" ?

Reply:
you can, i'll document this better.

This works `testwrapper ./... -v`.

Reply:
can you make it behave like go test instead? including with its `-args` support?

https://pkg.go.dev/cmd/go#hdr-Testing_flags and search for the two "For instance, the command" bits.

Reply:
Alternatively, don't even try to match and do something obviously different.

Reply:
PTAL